packages/mc-http-server/bin/start-mc.js

```diff
@@ -5,7 +5,12 @@ const fs = require('fs');
 const path = require('path');
 const shell = require('shelljs');
 const fetch = require('node-fetch');
-const replaceHtmlPlaceholders = require('@commercetools-frontend/mc-html-template/utils/replace-html-placeholders');
+const {
+  replaceHtmlPlaceholders,
+} = require('@commercetools-frontend/mc-html-template');
+const {
+  packageLocation: assetsPath,
```

Comment:
This is how the exported `__dirname` is useful

Reply:
Ah we need the string not the resolved thing. Got it.